### PR TITLE
appveyor.yml: example pypi upload

### PR DESCRIPTION
**Pull request by xoviat**
@boppreh 

This is not to be merged, but is an example of how to upload to pypi. I use git tag && git push --tags to do a new release on my packages. It's a few seconds of work.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+artifacts:
+  - path: chocolatey.log
+
+on_success:
+  - ps: |
+      If ($env:APPVEYOR_REPO_TAG -eq "true") {        
+        choco install pandoc
+        python setup.py sdist bdist_wheel
+        Invoke-Expression "twine upload -u $env:PYPI_USERNAME -p $env:PYPI_PASSWORD --repository-url https://upload.pypi.org/legacy/ --skip-existing dist/*" 2>&1 | Out-Null
+      }
```
Comment (xoviat) [inline appveyor.yml]: Here is what you are looking for.

Comment (xoviat) [inline appveyor.yml]: I should note that the pandoc is simply because I use setuptools-markdown.

**Comment by boppreh** [comment]
Funny, I just spent most of my day talking to the good people over PyPI and reporting bugs.

I'll take a look at this, thank you.

**Comment by xoviat** [comment]
I was looking at PEP 517 issues on distutils-sig and notice that you were having serious issues.

**Comment by boppreh** [comment]
Donald Stufft helped me identify the issues and workarounds, so it should be ok now. Unfortunately the `mouse` package won't have Windows installers because of PEP 517, but I think I can still upload them to the Github releases, like I do for https://github.com/boppreh/steamgrid/releases.

It's just a matter of small changes to [`make_release.py`](https://github.com/boppreh/keyboard/blob/master/make_release.py), which shouldn't be too hard.

Because this Pull Request is not meant for merging, I'm going to close it. But I'm thankful for it, and I'll keep it at hand as I update my release process.

**Comment by xoviat** [comment]
> Donald Stufft helped me identify the issues and workarounds, so it should be ok now. Unfortunately the mouse package won't have Windows installers because of PEP 517, but I think I can still upload them to the Github releases, like I do for https://github.com/boppreh/steamgrid/releases.

What does PEP 517 have to do with mouse? PEP 517 is about build system abstraction.

**Comment by boppreh** [comment]
Sorry, typo. I meant [PEP 527](https://www.python.org/dev/peps/pep-0527/), which deprecates certain file types to be uploaded to PyPI.

**Pull request closed without merging**